.github/ci-bionic/packages.apt

```diff
@@ -0,0 +1,11 @@
+pkg-config
```

Comment:
The file is already under `.github/ci`

https://github.com/ignitionrobotics/ign-common/blob/ign-common3/.github/ci/packages.apt